src/test.js

```diff
@@ -465,9 +465,11 @@ Test.prototype = {
 
 // Resets the fixture DOM element if available.
 function internalReset() {
```

Comment:
This code also belongs in reporter/html.js, which I think eliminates the need to define `id` twice. I like your [`testStart` suggestion](https://github.com/jquery/qunit/issues/974#issuecomment-201966604).